MdeModulePkg/Library/DxeCorePerformanceLib/DxeCorePerformanceLib.c

```diff
@@ -1047,7 +1047,13 @@ InsertFpdtRecord (
   switch (PerfId) {
     case MODULE_START_ID:
     case MODULE_END_ID:
-      GetModuleInfoFromHandle ((EFI_HANDLE)CallerIdentifier, ModuleName, sizeof (ModuleName), &ModuleGuid);
+      // MU_CHANGE [BEGIN] - CodeQL change
+      Status = GetModuleInfoFromHandle ((EFI_HANDLE)CallerIdentifier, ModuleName, sizeof (ModuleName), &ModuleGuid);

```

Comment:
`ModuleGuid` is an output of `GetModuleInfoFromHandle()`. Shouldn't the value be checked against `NULL` before being consumed further down?

```c
CopyMem (&FpdtRecordPtr.GuidEvent->Guid, &ModuleGuid, sizeof (FpdtRecordPtr.GuidEvent->Guid));
```

Reply:
Since it's a local, that's fine.